librz/util/buf.c

```diff
@@ -459,40 +697,91 @@ RZ_API bool rz_buf_append_buf_slice(RZ_NONNULL RzBuffer *b, RZ_NONNULL RzBuffer
 	return result;
 }
 
+/**
+ * \brief Append an array of bytes to the buffer.
+ * \param b A buffer with write permission.
+ * \param buf ...
+ * \param length ...
+ * \return Return the status of the operation.
+ *
+ * ...
+ */
 RZ_API bool rz_buf_append_bytes(RZ_NONNULL RzBuffer *b, RZ_NONNULL const ut8 *buf, ut64 length) {
 	rz_return_val_if_fail(b && buf && !b->readonly, false);
 
 	if (rz_buf_seek(b, 0, RZ_BUF_END) < 0) {
 		return false;
 	}
 
-	return rz_buf_write(b, buf, length) >= 0;
+	return rz_buf_write(b, buf, length) == length;
 }
 
+/**
+ * \brief Extend the size of the buffer.
+ * \param b A buffer with write permission.
+ * \param length The number of bytes to add at the end of the buffer.
+ * \return Return the status of the operation.
+ *
+ * ...
+ */
 RZ_API bool rz_buf_append_nbytes(RZ_NONNULL RzBuffer *b, ut64 length) {
 	rz_return_val_if_fail(b && !b->readonly, false);
 
 	return rz_buf_resize(b, rz_buf_size(b) + length);
 }
 
+/**
+ * \brief Add a ut16 number at the end of the buffer.
+ * \param b A buffer with write permission.
+ * \param n
+ * \return Return the status of the operation.
+ *
+ * ...
```

Comment:
remove these... I think it's fine for now. Maybe we should mention that the host endianess is used to store the bytes..

Reply:
I should remove the whole documentation or just `...`.

Reply:
just ...